Packages/com.unity.inputsystem/Documentation~/noisy-controls.md

```diff
@@ -1,15 +1,17 @@
-# Noisy Controls
+# Noisy controls
 
-The Input System can label a Control as "noisy". You can query this using the  [`InputControl.noisy`](../api/UnityEngine.InputSystem.InputControl.html#UnityEngine_InputSystem_InputControl_noisy) property.
+Noisy controls are those that can change value without any actual or intentional user interaction required. A good example of this is an accelerometer or gyroscope sensor in a cellphone. Even if the cellphone is at rest, there are usually fluctuations in the control's value readings. Another example are orientation readings from a head-mounted display.
```

Comment:
```suggestion
Noisy controls are those that can change value without any actual or intentional user interaction required. For example, they gyroscope sensor in a cellphone provides noisy input data because even if the cellphone is at rest, there are usually fluctuations in the control's value readings. Another example are orientation readings from a head-mounted display.
```